### ISSUE TITLE
Unable to open the site in Safari

### ISSUE THREAD

**Issue by kena0ki** [kena0ki/ai-font-gallery#1]
For the first time, the site opened correctly, but once I closed the browser and tried to reopen it, it no longer opened. 
Error message: "response served by service worker has redirections"

**Comment by kena0ki** [comment]
Webkit source code.
https://opensource.apple.com/source/WebCore/WebCore-7608.1.48/workers/service/context/ServiceWorkerFetch.cpp

**Comment by kena0ki** [comment]
The condition of the error from the above source code:
```
    if ((redirect != FetchOptions::Redirect::Follow || mode == FetchOptions::Mode::Navigate) && response.isRedirected())
        return ResourceError { errorDomainWebKitInternal, 0, response.url(), "Response served by service worker has redirections"_s, ResourceError::Type::AccessControl };
```

When we navigate from address bar, `mode` is "navigate" and `redirect` is "manual", so whether the error occur or not depends on `isRedirect`.
For the first request, we returned the preloaded response, but in Safari its `redirected` value was true, thus the error occurred.
The `redirected` value of the response from the server is false and we don't need to use the preload response since the cache is used for the large files (.onnx, .wasm) rather than PWA, so we can simply remove the logic for preload.

**Comment by kena0ki** [comment]
Fixed in 790b736a0de4ae62e384c736559e3181c05761d8
